python-lib/dku_azure/clusters.py

```diff
@@ -151,6 +155,10 @@ def build(self):
         if self.private_access:
             cluster_params["api_server_access_profile"] = self.private_access
 
+        if self.custom_config:
+            custom_config_dict = json.loads(self.custom_config)
+            cluster_params = _merge_objects(cluster_params, custom_config_dict)
```

Comment:
I tried setting 
```
{
"kubernetes_version":"1.21"
}
```
as a custom json, it did...not end well (I think resulting version was set to `1.221.21`)
My guess is that's because `string` is Iterable in Python, so we fall in the first `elif` in `_merge_objects`
(_If_ you fix it (not sure it's worth it), we'll also need to fix it in the other two plugins...)

Reply:
let's not do not requested stuff that would potentially break everything :wink: